celsius_farenheit/convert_to_celsius.c

```diff
@@ -0,0 +1,22 @@
+
+#include <stdio.h>
+
+int main()
+{
+    float celsius,fahr;
+    int lower, upper, step;
+
+    lower = 0;
+    upper = 300;
+    step =20;
+    printf("%4s %7s\n", "fahr","celsius");
+    fahr=lower;
+    while(fahr <= upper){
+        celsius = (5.0/9.0)*(fahr - 32.0);
+        printf("%4.0f %7.1f\n",fahr,celsius);
```

Comment:
what do `4.0` and `7.1` mean?

Reply:
4.0f : 
1 argument of printf function is float, 4- 4 characters in whole part, .0 - 0 characters in decimal part to print

Reply:
got it.